quickstart/examples/inference-scheduling/gaie-inference-scheduling/values.yaml

```diff
@@ -5,20 +5,21 @@ inferenceExtension:
     ###################
     name: llm-d-inference-scheduler
     hub: ghcr.io/llm-d
-    tag: v0.2.1
+    tag: v0.3.0-rc.1
     ###################
     # name: epp
     # hub: registry.k8s.io/gateway-api-inference-extension
-    # tag: v0.5.1
+    # tag: v1.0.0-rc.3
```

Comment:
use rc.4. it will be ready in soon and includes the helm chart default port for the required service